pkg/cmd/opts/version.go

```diff
@@ -84,7 +84,7 @@ func (o *CommonOptions) GetPackageVersions(ns string, helmTLS bool) (map[string]
 				if v != "" {
 					switch i {
 					case 0:
-						table.AddRow("kubectl", info(v))
+						table.AddRow("kubectl (installed in JX_BIN)", info(v))
```

Comment:
+1, an important distinction :-)

Reply:
I saw some confusion from new users in the community slack channel, so decided to make it clearer :)